flink-doris-connector/src/main/java/org/apache/doris/flink/backend/BackendClient.java

```diff
@@ -69,27 +69,23 @@ public BackendClient(Routing routing, DorisReadOptions readOptions) {
     private void open() {
         logger.debug("Open client to Doris BE '{}'.", routing);
         TException ex = null;
-        for (int attempt = 0; !isConnected && attempt < retries; ++attempt) {
+        for (int attempt = 0; attempt < retries; ++attempt) {
             logger.debug("Attempt {} to connect {}.", attempt, routing);
             TBinaryProtocol.Factory factory = new TBinaryProtocol.Factory();
             transport = new TSocket(routing.getHost(), routing.getPort(), socketTimeout, connectTimeout);
             TProtocol protocol = factory.getProtocol(transport);
             client = new TDorisExternalService.Client(protocol);
-            if (isConnected) {
-                logger.info("Success connect to {}.", routing);
-                return;
-            }
             try {
                 logger.trace("Connect status before open transport to {} is '{}'.", routing, isConnected);
                 if (!transport.isOpen()) {
                     transport.open();
                     isConnected = true;
```

Comment:
maybe add log `logger.info("Success connect to {}.", routing);` in this be better?

Reply:
> maybe add log `logger.info("Success connect to {}.", routing);` in this be better?

okay, thanks for your review

Reply:
> maybe add log `logger.info("Success connect to {}.", routing);` in this be better?

updated